README.md

```diff
@@ -1,68 +1,129 @@
-# sudoers
-> An Ansible role for comprehensive management of a sudoers configuration.
-
-[![Build Status](https://travis-ci.org/wtcross/ansible-sudoers.svg?branch=master)](https://travis-ci.org/wtcross/ansible-sudoers)
+# ahuffman.sudoers
+An Ansible role for configuring the /etc/sudoers file and /etc/sudoers.d files.
 
 This role makes it possible to completely define your sudoers configuration with Ansible. All of the following are configurable:
 - defaults
 - aliases
+  * Users
+  * Runas
+  * Hosts
+  * Commands
 - specifications
 
-*Tip:* Here's a [great document about sudoers configuration](https://help.ubuntu.com/community/Sudoers)
```

Comment:
Is there a reason you removed this note? I don't care about the source, but wanted the README in this role to point to some comprehensive sudoers guide. Open to suggestions on an alternative.